webapp/src/Components/Pages/HomePage.tsx

```diff
@@ -44,65 +45,78 @@ class HomePageComponent extends React.PureComponent<Props, State> {
         getLoggedInUser()
             .then((loggedInUser) => this.props.setLoggedInUser(loggedInUser))
             .catch(() => undefined)
     }
 
     public render() {
-        const {classes, width, loggedInUser} = this.props
+        const { classes, width, loggedInUser } = this.props
 
-        const alignCenterProps: GridProps = {container: true, justify: "center", alignItems: "center"}
+        const alignCenterProps: GridProps = { container: true, justify: "center", alignItems: "center" }
         return (
             <div className={classes.backgroundContainer}>
                 <UploadModalWrapper buttonStyle="floating">
                     <div className={classes.root}>
                         <Grid container justify="center" alignItems="flex-start" spacing={40} className={classes.child}>
-                            <Grid item xs={12} {...alignCenterProps} style={{minHeight: "300px"}} direction="column">
-                                <Logo imgStyle={{maxWidth: "80vw"}}/>
-                                {this.state.replayCount &&
-                                <>
-                                    <br/>
-                                    <Typography>
-                                        <i>{this.state.replayCount} replays and counting...</i>
-                                    </Typography>
-                                </>
-                                }
+                            <Grid item xs={12} {...alignCenterProps} style={{ minHeight: "300px" }} direction="column">
+                                <Logo imgStyle={{ maxWidth: "80vw" }} />
+                                {this.state.replayCount && (
+                                    <>
+                                        <br />
+                                        <Typography>
+                                            <i>{this.state.replayCount} replays and counting...</i>
+                                        </Typography>
+                                    </>
+                                )}
                             </Grid>
-                            <Grid item xs={11} {...alignCenterProps} style={{padding: "20px 0 20px 0"}}>
-                                <Search usePaper/>
+                            <Grid item xs={11} {...alignCenterProps} style={{ padding: "20px 0 20px 0" }}>
+                                <Search usePaper />
                             </Grid>
-                            <Grid item xs={12} sm={10} md={8} container spacing={16} alignItems="center"
-                                  style={{maxWidth: 550}}>
-                                <Grid item xs={6} style={{textAlign: "center"}}>
-                                    {loggedInUser ?
-                                        <Link to={PLAYER_PAGE_LINK(loggedInUser.id)} style={{textDecoration: "none"}}>
+                            <Grid
+                                item
```

Comment:
personally i find the old way better is there a way to make the formatter combine lines a little bit?

Reply:
Currently, the only way to ignore lines like these is to tell the formatter to ignore the line using `// prettier-ignore` sprinkled everywhere. This is, however, less than ideal. For readability purposes, we tell the formatter to never let our files exceed the 120th column. It might not hurt to set the indentation for every file to 2 spaces instead of 4.